Lab_1 (1).ipynb

```diff
@@ -0,0 +1,948 @@
+   "source": [
+    "s = np.random.poisson(lam = 100, size = 10000) # draw samples\n",
+    "count, bins, ignored = plt.hist(s, 20)\n",
+    "plt.show() # plot histogram"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bdfb7ff8-138c-4be9-aad0-5c87585cf5e6",
+   "metadata": {},
+   "source": [
+    "The lambda value in the Poisson distribution corresponds to the mean. In each plot it is clear that the lambda value is roughly in the center of each distribution. As I increased the sample size the distribution appeared to become more symmetric. The variance increases as lambda increases as well because it is equal to the square root of lambda."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a7beed98-d27b-4983-b5f0-9fd462489e29",
+   "metadata": {},
+   "source": [
+    "# What is the probability that the background produced a signal as signal-like or more than what I observed? "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 119,
+   "id": "9adccc2f-23ad-4721-b6aa-7f497498e31a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "val = 120\n",
+    "lam = 100\n",
+    "\n",
+    "frozen_poisson = stats.poisson(lam)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 120,
+   "id": "e9392abe-3458-4497-9fc4-11ff4e1f2020",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.02266932907835273\n"
+     ]
+    }
+   ],
+   "source": [
+    "probability = frozen_poisson.cdf(val)\n",
+    "print( 1- probability)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 121,
+   "id": "860b05a4-6d8a-4425-bad0-e7c290443486",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2.0014988446547286\n"
+     ]
+    }
+   ],
+   "source": [
+    "sigma = stats.norm.ppf(probability)\n",
+    "print(sigma)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "79eba8bd-3e62-4f6d-8983-aec60ea87831",
+   "metadata": {},
+   "source": [
+    "The mean of a Poisson distribution can be a non-integer value, but that does not mean the expected value will be non-integer, it will just be the integer closest to the mean. Even though each event is discrete, we can end up with non-integer parameters because the amount of occurences of the events. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "48cd79e9-e470-4171-8a34-f08bcba06230",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.8"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
```

Comment:
Overall it's a little wordy for a python document and I'd adjust the formatting slightly (ie reducing the number of cells so there aren't as many code blocks to parse). Please let me know if you'd like me to review anything further, but overall it's a very solid lab.